screens/Home/MyVcsTab.tsx

```diff
@@ -152,6 +154,13 @@ export const MyVcsTab: React.FC<HomeScreenTabProps> = (props) => {
         error={storeErrorTranslationPath}
         onDismiss={controller.DISMISS}
       />
+      <MessageOverlay
+        isVisible={controller.isBindingError}
+        title={controller.walletBindingError}
+        onCancel={() => {
+          controller.DISMISS;
+        }}
```

Comment:
this can be written in a more compact format

```suggestion
        onCancel={controller.DISMISS}
```